build-tools/installers/create-installers.targets

```diff
@@ -132,7 +133,7 @@
   </ItemGroup>
   <ItemGroup>
     <_MSBuildFilesWin Include="$(MSBuildSrcDir)\libzip.dll" />
-    <_MSBuildFilesWin Include="$(MSBuildSrcDir)\x64\libzip.dll" />
+    <_MSBuildFilesWin Include="$(MSBuildSrcDir)\lib64\libzip.dll" />
```

Comment:
I didn't know a `lib64` directory would work, does it matter if we have other files in `x64`?

Reply:
LibZipSharp has been setup to use `x64` see https://github.com/xamarin/LibZipSharp/blob/master/Native.cs#L414

So it has to be in that directory. What other files do we have in x64?

Reply:
Also it makes it compatible with linux as well as that will look in `lib64` by default (if it exists)

Reply:
I thought we had `libwinpthread-1.dll` in `x64`, but we don't. I guess we might eventually have others.